stacks/api/bucket/bucket.schema.resolver.spec.ts

```diff
@@ -3,7 +3,7 @@ import {DatabaseTestingModule, DatabaseService, ObjectId} from "@spica-server/da
 import {PreferenceModule, Preference} from "@spica-server/preference";
 import {SchemaModule} from "@spica-server/core/schema";
 import {BucketSchemaResolver} from "./bucket.schema.resolver";
-import {BucketService} from "./bucket.service";
+import {BucketService} from "./services/bucket.service";
```

Comment:
We need to change this import clause to import the reference from `@spica-server/bucket/services`